pages/dkp/kommander/2.2/dkp-upgrade/index.md

```diff
@@ -13,12 +13,12 @@ The DKP upgrade represents an important step of your environment's lifecycle, as
 
 Verified upgrade paths are as follows:
 
-| To    | From | 1.8.4 | 1.8.5 | 2.1.0 | 2.1.1 |
-|-------|------|:-----:|:-----:|:-----:|:-----:|
-| 1.8.5 |      | Yes   |       |       |       |
-| 2.1.0 |      | Yes   | Yes   |       |       |
-| 2.1.1 |      | Yes   | Yes   | No    |       |
-| 2.2.0 |      | No    | No    | Yes   | Yes   |
+| From  | 1.8.5 | 2.1.0 | 2.1.1 | 2.2.0 |
```

Comment:
I also like this table as an alternative 🤔 

| From  |  To   |       |       |
| ----- | :---: | :---: | :---: |
| 1.8.4 | 1.8.5 | 2.1.0 | 2.1.1 |
| 1.8.5 | 2.1.0 | 2.1.1 |       |
| 2.1.0 | 2.2.0 |       |       |
| 2.1.1 | 2.2.0 |       |       |

Reply:
This one is actually more straightforward to me. does markdown support merging the column head only? if so this would be perfect.

Reply:
yeah this one seemed easier to read to me as well. Markdown doesn't support that, I played with using some HTML snippets in the table for a bit but was unsuccessful (the styling was a bit off on all my attempts). Its definitely possible but not sure if we would have to redo the entire table in HTML.

Reply:
or insert an empty column to help visually divide between From and To?

Reply:
what do you think about the latest change?

<img width="907" alt="image" src="https://user-images.githubusercontent.com/27781468/162498808-86e9d5b9-67eb-402d-8c97-9d095f4eb2e8.png">

Reply:
This is another option

<img width="921" alt="image" src="https://user-images.githubusercontent.com/27781468/162500329-98d6d5f3-d65a-4fda-bfbc-c1a93eb684bf.png">

Reply:
a compatibility matrix would be ideal. but what about 

| From  |  To   |   
| ----- | :---: | 
| 1.8.4 | 1.8.5 , 2.1.0 , 2.1.1 |
| 1.8.5 | 2.1.0, 2.1.1        |
| 2.1.0 | 2.2.0      |
| 2.1.1 | 2.2.0        |

as i user i only care the version i am currently ON and i want to see every version i can upgrade TO as easily as i can. I dont need to grok the entire compatibility matrix. 

feel free to ignore this, this is from the perspective of a developer and i couldnt be farther from user perspective :see_no_evil:

Reply:
I actually like that table the best out of all the other ones I looked at!

Reply:
I need to have @kevinmiller-conley or @msdolbey take a look.

Reply:
Guys, I have to disagree here. I am not a fan of the dots, and the commas are very hard to read as well. We need to stick to the Yes/No model. I'm ok rearranging the versions though.

Reply:
just to clarify, we can also use the versions in each cell.